modules/manage/pages/kubernetes/k-schema-controller.adoc

```diff
@@ -0,0 +1,272 @@
+
+You can find all configuration options for the Schema resource in the xref:reference:k-crd.adoc#k8s-api-github-com-redpanda-data-redpanda-operator-operator-api-redpanda-v1alpha2-schema[CRD reference].
+
+.`schema.yaml`
+[source,yaml]
+----
+apiVersion: cluster.redpanda.com/v1alpha2
+kind: Schema
+metadata:
+  name: <subject-name> <.>
+  namespace: <namespace> <.>
+spec:
+  cluster:
+    clusterRef:
+      name: <cluster-name> <.>
+  schemaType: avro <.>
+  compatibilityLevel: Backward <.>
+  references: [] <.>
+  text: | <.>
+    {
+      "type": "record",
+      "name": "test",
+      "fields": [
+        { "type": "string", "name": "field1" },
+        { "type": "int", "name": "field2" }
+      ]
+    }
+----
+
+<.> *Subject name*: The name of the subject for the schema. When data formats are updated, a new version of the schema can be registered under the same subject, enabling backward and forward compatibility.
+<.> *Namespace*: The namespace in which to deploy the Schema resource. The Schema resource must be deployed in the same namespace as the Redpanda resource defined in `clusterRef.name`.
+<.> *Cluster name*: The name of the Redpanda resource that defines the Redpanda cluster to which you want to upload the schema.
+<.> *Compatibility level*: Defines the compatibility level for the schema. Options are `Backward` (default), `BackwardTransitive`, `Forward`, `ForwardTransitive` `Full`, `FullTransitive`, or `None`. See <<Choose a compatibility mode>>.
+<.> *Schema type*: Specifies the type of the schema. Options are `avro` (default) or `protobuf`. For JSON Schema, include `"$schema":` in the `text` to indicate the JSON Schema draft version. See <<Choose a schema type>>.
+<.> *References*: Any references you want to add to other schemas. If no references are needed, this can be an empty list (default). See <<Use schema references>>.
+<.> *Schema body*: The body of the schema, which defines the data structure.
+
+=== Choose a schema type
+
+Redpanda's Schema Registry supports the following schema types:
+
+* *Avro*: A widely used serialization format in event-driven architectures.
+* *Protobuf*: Popular for defining data structures in gRPC APIs and efficient data serialization.
+* *JSON Schema*: Dynamic, schema-based validation for JSON documents.
+
+If no type is specified, Redpanda defaults to Avro.
+
+=== Choose a compatibility mode
+
+Compatibility modes determine how schema versions within a subject can evolve without breaking existing data consumers. Redpanda supports the following compatibility levels:
+
+* `None`: Disables compatibility checks, allowing any schema change.
+* `Backward`: New schemas are backward-compatible with previous versions, ensuring that new consumers can read data created with older schemas.
+* `BackwardTransitive`: Enforces backward compatibility across all versions, not just the latest.
+* `Forward`: New consumers can process data written with older schemas, allowing forward compatibility.
```

Comment:
`Backward` and `Forward` sound the same - let me know if I misunderstand. Should the `Forward` description match the one from here? https://deploy-preview-848--redpanda-docs-preview.netlify.app/24.3/manage/schema-reg/schema-reg-api/#configure-schema-compatibility

Reply:
thanks for calling this out